tasks/agent.py

```diff
@@ -159,6 +159,14 @@ def build(
     """
     flavor = AgentFlavor[flavor]
 
+    if python_runtimes:
+        print(
+            color_message(
+                '--python-runtimes is deprecated and will always be 3. Please remove this parametes from your scripts',
```

Comment:
```suggestion
                'Warning: --python-runtimes is deprecated and will always be 3. Please remove this parameter from your scripts',
```

Reply:
Added the `Warning` prefix in the helper, and fixed the typo there!